cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -870,12 +871,25 @@ def make_resource(self, data, **kwargs):
         return AmiSearchFilters(**data)
 
 
+class TimeoutsSchema(BaseSchema):
+    """Represent the schema of the Timeouts section."""
+
+    head_node_bootstrap_timeout = fields.Int(metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
+    compute_node_bootstrap_timeout = fields.Int(metadata={"update_policy": UpdatePolicy.COMPUTE_FLEET_STOP})
```

Comment:
Shall we at least enforce that the timeout is > 0?

Reply:
Added min=1